config/prometheus/prometheus_rules.yaml

```diff
@@ -24,3 +24,41 @@
       "for": "5m"
       "labels":
         "severity": "critical"
+  - "name": "thin-pool-alert.rules"
+    "rules":
+    - "alert": "ThinPoolDataUsageAtThresholdNearFull"
+      "annotations":
+        "description": "Thin pool in the VolumeGroup is nearing full. Data deletion or thin pool expansion is required."
+        "message": "Thin Pool data utilization in the VolumeGroup {{ $labels.device_class }} has crossed 75 % on node {{ $labels.node }}. Free up some space or expand the thin pool."
+      "expr": |
+        topolvm_thinpool_data_percent > 75.00  and topolvm_thinpool_data_percent < 85.00
+      "for": "5m"
+      "labels":
+        "severity": "warning"
+    - "alert": "ThinPoolDataUsageAtThresholdCritical"
+      "annotations":
+        "description": "Thin pool in the VolumeGroup is critically full. Data deletion or thin pool expansion is required."
+        "message": "Thin Pool data utilization in the VolumeGroup {{ $labels.device_class }} has crossed 85 % on node {{ $labels.node }}. Free up some space or expand the thin pool immediately."
```

Comment:
Was this generated by the jsonnet? I don't see any changes to the libsonnet file.